src/NuGet.Clients/PackageManagement.VisualStudio/Telemetry/ProjectTelemetryEvent.cs

```diff
@@ -7,7 +7,7 @@ namespace NuGet.PackageManagement.Telemetry
 {
     public abstract class ProjectTelemetryEvent
     {
-        public ProjectTelemetryEvent(string nuGetVersion, Guid projectId)
+        public ProjectTelemetryEvent(string nuGetVersion, string projectId)
```

Comment:
It's the project id only. Earlier we use to pass the Guid and while emitting event, convert it to string. But now we just take project id as string.